### PR TITLE
Web serving hotfix

### PR DESCRIPTION
**Pull request by altostratous**
Just rebased #116   
*  Make percentile latency limits more tight, 
*  Add a message to client connecting retrial.

### CODE REVIEW

File: benchmarks/web-serving/faban_client/bootstrap.sh [hunk 1]
```diff
@@ -14,7 +14,8 @@ WEB_SERVER_IP=$1
 LOAD_SCALE=${2:-7}
 
 while [ "$(curl -sSI ${WEB_SERVER_IP}:8080 | grep 'HTTP/1.1' | awk '{print $2}')" != "200" ]; do
-  sleep 1
+    echo "Cannot connect to web_server container at: ${WEB_SERVER_IP}:8080"
```
Comment (rahulketch) [inline benchmarks/web-serving/faban_client/bootstrap.sh]: It's not necessary that this message is displayed when we cannot connect to the web_server. We also get this when we are able to connect but the HTTP response is something other than 200.  Maybe we can write the message accordingly.

File: benchmarks/web-serving/faban_client/Dockerfile [hunk 2]
```diff
@@ -9,9 +9,8 @@ RUN apt-get update && apt-get install -y \
 
 # Setup Faban 
 
-RUN wget http://faban.org/downloads/faban-kit-latest.tar.gz
-RUN tar zxvf faban-kit-latest.tar.gz
-
+RUN wget http://faban.org/downloads/faban-kit-1.4.tar.gz
```
Comment (rahulketch) [inline benchmarks/web-serving/faban_client/Dockerfile]: should we add faban version as an environment variable like we do in case of web-search client?

Comment (neo-apz) [reply benchmarks/web-serving/faban_client/Dockerfile]: Yes please do it like in web-search.
also, fix the java version.

File: benchmarks/web-serving/faban_client/bootstrap.sh (hunk 1 above)
Comment (neo-apz) [reply benchmarks/web-serving/faban_client/bootstrap.sh]: We can say something like "Bad status code. Retrying to connect to the server..."

File: benchmarks/web-serving/faban_client/Dockerfile (hunk 2 above)
Comment (altostratous) [reply benchmarks/web-serving/faban_client/Dockerfile]: I think it is covered in #143

Comment (neo-apz) [reply benchmarks/web-serving/faban_client/Dockerfile]: @altostratous can you please fix the java version, so it won't break in future.

Comment (altostratous) [reply benchmarks/web-serving/faban_client/Dockerfile]: Yes, I fixed it.

Comment (altostratous) [reply benchmarks/web-serving/faban_client/Dockerfile]: Yes, I fixed it.

**Pull request merged**